src/Presentation/Console/Commands/InitSdkCommand.php

```diff
@@ -78,162 +68,50 @@ public function __construct(
     protected function configure()
     {
         parent::configure();
-        $this->createDatabase();
 
-        $settings = $this->readSettingDefinitions();
+        $settings = $this->yamlParser->parseFile($this->settingsPath)['settings'];
```

Comment:
Please do not use array access in that way to avoid possible exceptions.
Use the following way instead to be on the safe side.
```
$configuration = $this->yamlParser->parseFile($this->settingsPath);
$settings = $configuration['settings'] ?? [];

Reply:
pls check the logic. it's always exist